src/main/resources/network/brightspots/rcv/hints_winning_rules.txt

```diff
@@ -24,6 +24,8 @@ Continue until Two Candidates Remain: Single-winner ranked-choice voting electio
 
 First Round Determines Threshold: Single-winner ranked-choice voting elections can use the first round to determine the winning threshold, rather than the last round. This will not affect the eventual winner. This may elect a winner that does not reach the threshold.
 
+Prevent Final Round From Having Just One Candidate: If two candidates remain and neither has met the threshold, select one as the winner instead of running an elimination round.
```

Comment:
I'm curious what @chughes297 thinks here. My instinct is that we would want the default behavior to be that we prevent the final round from having just one candidate. And if that's the case, perhaps this config setting should be "Allow Final Round to Have Just One Candidate" and it should default to false.

Also, we should clarify which types of contests this is relevant for. (And disable it in the UI/raise an error when it's used inappropriately.)

Reply:
I defer to @chughes297 on what the default behavior should be.

I don't know if any legislation specifies this explicitly, so it seems that it comes down to which statute is seen more frequently:
1. Tabulate until somebody crosses the threshold
2. Tabulate until two candidates remaining, then pick the top vote-getter

In the first case, that would indicate that we _should_ go down to one candidate. In the second case, we _shouldn't_.



As for clarifying contest types: do you mean disabling this option for STV, or are there other cases where it should be disabled?

Reply:
Agree that we should default to including two candidates in the final round. As I look at this now though I'm not sure we want to give any users an option here - the default tabulation rule of having two people in the final round is all we should do, imo. I don't think there's any value in giving people the option of going to a single-candidate final round (and I can't think of a law that requires it? Sorry if I gave confusing input earlier that implied that...)